tests/manifests/vm-fedora-template.yml

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
```

Comment:
You can modify /tests/manifests/virt-testing-vm.yml
And add the cloud init part (it parameter anyway) 
no need for new file.

Reply:
/tests/manifests/virt-testing-vm.yml - is template for VMI, but in my tests I need to create VM (for verifying start/stop.. etc)

may be it's better to rename virt-testing-vm.yml to virt-testing-vmi.yml because its name is a little confusing.. and rename my vm-fedora-template.yml to virt-testing-vm.yml for future needs

Reply:
I updated virt-testing-vm.yml, now it creates VM instead of VMI. 
Also, had to update high_performance_vm_test.go